app/models/loan_question.rb

```diff
@@ -137,7 +137,7 @@ def active?
   end
 
   def child_groups
-    children.select(&:group?)
+    children.order(:position).select(&:group?)
```

Comment:
Please avoid doing AR scopes on the children method because it forces a cascading series of db queries. We are using the closure_tree full-tree eager loading functionality but that only works if you keep everything in memory. I have fixed this as I noticed another spot where it also needed fixing. Need to override children to make it not possible to do this in future. Will likely do that in an upcoming refactor.